core/src/test/scala/org/apache/spark/sql/BaseTiSparkTest.scala

```diff
@@ -92,15 +92,15 @@ class BaseTiSparkTest extends QueryTest with SharedSQLContext {
   }
 
   protected def supportTTLUpdate: Boolean = {
-    StoreVersion.minTiKVVersion("3.0.5", this.ti.tiSession.getPDClient)
+    StoreVersion.minTiKVVersion(Version.RESOLVE_LOCK_V3, this.ti.tiSession.getPDClient)
   }
 
   protected def supportBatchWrite: Boolean = {
     // currently only the following versions support BatchWrite
     // 3.0.x (x >= 14)
     // 3.1.x (x >= 0)
-    StoreVersion.minTiKVVersion("3.0.14", this.ti.tiSession.getPDClient) && !StoreVersion
-      .minTiKVVersion("4.0.0", this.ti.tiSession.getPDClient)
+    StoreVersion.minTiKVVersion(Version.BATCH_WRITE, this.ti.tiSession.getPDClient) && !StoreVersion
```

Comment:
add a TODO that batch write on v4.0.0 is not yet ready?

Reply:
it's just a code refactor!

Reply:
i'm working on v4.0.0, so it's not necessary to add a `TODO`.